leaf-ui/rappid-extensions/BackboneModels.js

```diff
@@ -1 +1,29 @@
-/** This file contains backbone model representations of the original model objects - WIP */
\ No newline at end of file
+/** This file contains backbone model representations of the original model objects - WIP */
+
+
+// FuncSegmentModel from FuncSegment class in modelObjects.js
+var FuncSegmentModel = Backbone.Model.extend({
+    idAttribute: "uid",
+
+    // initialize function - could also save all of the input parameters into one variable 
+    // 'options' and then use that like Yesuegen 
+    initialize: function (options) {
+        this.funcType = options.funcType;
```

Comment:
Can we change `funcX` to `refEvaluationValue`.
We are going to take func out of each of these names.
`type`, `refEvaluationValue`, `startTP`, `stopTP`.